guides/v2.2/cloud/project/project-multi-sites.md

```diff
@@ -8,157 +8,246 @@ functional_areas:
 
 After successfully creating and testing the local installation to use multiple stores, you must prepare your Integration environment:
 
-1.  **Configure routes**—specify how incoming URLs are handled by {{site.data.var.ee}}
-1.  **Set up websites, stores, and store views**—configure using the {{site.data.var.ee}} Admin panel
-1.  **Modify `magento-vars.php` file**—specify the values of the `MAGE_RUN_TYPE` and `MAGE_RUN_CODE` variables
-1.  **Deploy**—deploy and test the `integration` branch
+1. **Configure routes or locations**—specify how incoming URLs are handled by {{site.data.var.ee}}
+   -  [Routes for separate domains](#routes)
+   -  [Locations for shared domains](#locations)
+1. **Set up websites, stores, and store views**—configure using the {{site.data.var.ee}} Admin UI
+1. **Modify Magento variables**—specify the values of the `MAGE_RUN_TYPE` and `MAGE_RUN_CODE` variables in the `magento-vars.php` file
+1. **Deploy**—deploy and test the `integration` branch
 
-### Configure routes
+### Configure routes for separate domains {#routes}
 
-Magento Enterprise Edition *routes* define how incoming URLs are processed. The way you configure routes depends on how you want your site to operate. We suggest configuring routes for integration as follows. You can edit the values later if your needs change.
+Routes define how to process incoming URLs. Multiple stores with unique domains requires you to define each domain in the `routes.yaml` file. The way you configure routes depends on how you want your site to operate.
 
-{:.bs-callout .bs-callout-info}
+{:.bs-callout-info}
 For Pro, you must create a [Support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to set up routes in the Staging or Production environment.
 
 #### To configure routes in an integration environment:
 
-1.  On your local workstation, log in as the [Magento file system owner]({{ page.baseurl }}/cloud/before/before-workspace-file-sys-owner.html).
-
-1.  Change to your {{site.data.var.ee}} base directory.
-
-1.  Open the `.magento/routes.yaml` file in a text editor.
-
-1.  Replace the contents with the following:
-
-    ```yaml
-    "http://{default}/":
-        type: upstream
-        upstream: "mymagento:php"
-
-    "https://{default}/":
-        type: upstream
-        upstream: "mymagento:php"
-
-    "http://*.{default}/":
-        type: upstream
-        upstream: "mymagento:php"
-
-    "https://*.{default}/":
-        type: upstream
-        upstream: "mymagento:php"
-    ```
-
-1.  Save your changes to the `routes.yaml` file.
+1. On your local workstation, log in as the [Magento file system owner]({{ page.baseurl }}/cloud/before/before-workspace-file-sys-owner.html).
+
+1. Change to your {{site.data.var.ee}} base directory.
+
+1. Open the `.magento/routes.yaml` file in a text editor.
+
+1. Replace the contents with the following:
+
+   ```yaml
+   "http://{default}/":
+       type: upstream
+       upstream: "mymagento:http"
+
+   "https://{default}/":
```

Comment:
right now, routes with https protocol will be added by default if you have the same for http, so no needs to add them. Probably we need to add this info where we describe how to work with routes. And add link to routes description in this article too.